java/runtime-common/src/main/java/org/ray/spi/RemoteFunctionManager.java

```diff
@@ -53,10 +52,9 @@
   void unregisterApp(UniqueID driverId);
 
   /**
-   * load resource and functions for this driver this function is used by the workers on demand when
-   * a required function is not found in {@code LocalFunctionManager}.
+   * load resource.
    */
-  LoadedFunctions loadFunctions(UniqueID driverId);
+  ClassLoader loadResource(UniqueID driverId);
```

Comment:
If the purpose of this function is to get the class loader for the driver. maybe we'd better rename it to `getClassLoader`. `loadResource` sounds ambiguous.